kamelet-support/src/test/java/io/kaoto/backend/metadata/parser/step/camelroute/CamelRouteParseCatalogTest.java

```diff
@@ -140,13 +140,25 @@ private void assertBrowseJsonHasBeenParsedCorrectly(
         );
     }
 
+    @Test
+    void checkDuration() {
+        InMemoryCatalog<Step> catalog = new InMemoryCatalog<>();
+        ParseCatalog<Step> camelParser = parseCatalog.getParser(CAMEL_ZIP);
+        assertTrue(catalog.store(camelParser.parse().join()));
+        Step consumer = catalog.searchByID("timer-consumer");
+        assertTrue(consumer.getParameters().stream().parallel()
+                .anyMatch(p -> p.getId().equalsIgnoreCase("delay") && p instanceof StringParameter));
+        Step activemq = catalog.searchByID("activemq-action");
+        assertTrue(activemq.getParameters().stream().parallel()
+                .anyMatch(p -> p.getId().equalsIgnoreCase("deliveryDelay") && p instanceof NumberParameter));
```

Comment:
the check of the `long` doesn't match the name of the test. i think it would better to provide 2 different tests

Reply:
and realizing that there was also the `numeric` which was added itn he same PR; So better with 3 tests

Reply:
Changed the name of the function.